config/config.go

```diff
@@ -57,6 +57,7 @@ const (
 
 	EnvKubernetesBurst         = "KUBERNETES_BURST"
 	EnvKubernetesQPS           = "KUBERNETES_QPS"
+	EnvKubernetesCacheEnabled  = "KUBERNETES_CACHE_ENABLED"
```

Comment:
You should probably document this in the README.adoc - see the section in there on all the environment variables. Add the new ones in there.

Reply:
+1
Yes, thatnks for the pointer.